### PR TITLE
Update gem versions, rubocop configuration plus minor changes

### PR DESCRIPTION
**Pull request by tferrerm**
## Summary

- Ruby is updated to version 2.6.4
- Rubocop is upgraded and more rubocop extensions are added
- `therubyracer` is replaced with replaced with `mini_racer` as `therubyracer` no longer seems to work.
- `rails-erd` is added (used to generate ER diagrams)
- `test-prof` is added
- `rails_helper.rb` is modified so when the `PRINT_SQL` environment variable is defined, database operations will be logged to `stdout` when testing.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 
 # Use postgresql as the database for Active Record
-gem 'pg', '~> 1.1'
+gem 'pg', '~> 1.1.0'
 # Use Puma as the app server
-gem 'puma', '~> 3.12'
+gem 'puma', '~> 3.12.0'
 
-gem 'therubyracer', platforms: :ruby
+gem 'mini_racer'
```
Comment (lucasVoboril) [inline Gemfile]: :eyes: Can you tag this gem version :pray:  in Gemfile.lock says we are using `0.2.6`

File: .rubocop.yml [hunk 2]
```diff
@@ -11,11 +11,12 @@ AllCops:
...
 
-Documentation:
+Style/Documentation:
   Enabled: false
 
-LineLength:
+Metrics/LineLength:
   Max: 99
 
 HasAndBelongsToMany:
```
Comment (holywyvern) [inline .rubocop.yml]: probably `Rails/HasAndBelongsToMany`, not a big deal.

File: Gemfile [hunk 3]
```diff
@@ -59,8 +61,10 @@ group :development, :test do
   gem 'faker'
 
   # Lints
-  gem 'rubocop', '~> 0.65.0', require: false
-  gem 'rubocop-rspec', '~> 1.32'
+  gem 'rubocop', '~> 0.75.0', require: false
+  gem 'rubocop-performance', '~> 1.4.0'
```
Comment (holywyvern) [inline Gemfile]: this probably all should be `require: false too`

**Review by blacksam07 — APPROVED** [review]

**Comment by tferrerm** [comment]
Most of these changes have been included in other PRs or merged already.

**Pull request closed without merging**